app/src/androidTest/java/com/troido/bless/app/base/EnabledBluetoothTestCase.kt

```diff
@@ -0,0 +1,30 @@
+package com.troido.bless.app.base
+
+import androidx.test.platform.app.InstrumentationRegistry
+import com.troido.bless.app.utils.BluetoothStateChanger
+import org.junit.Before
+
+/**
+ * Base test class that enables bluetooth before any of the test methods will be executed
+ */
+abstract class EnabledBluetoothTestCase {
+    companion object{
+        private var testClass: Class<out EnabledBluetoothTestCase?>? = null
+
+    }
+
+    @Before
+    fun setUp() {
+        println("Before in base test class")
+
+        if (this.javaClass.equals(testClass)) {
+            return;
+        }
+
+        val bluetoothStateChanger =
+            BluetoothStateChanger(true, InstrumentationRegistry.getInstrumentation().targetContext)
+        bluetoothStateChanger.process()
+
+        testClass = this.javaClass
+    }
+}
```

Comment:
There are no new lines at the end of the file including gradle files. Could you please change it?

Reply:
I'll add ktlint plugin which formats the code automatically